pex/resolve/locker.py

```diff
@@ -201,6 +211,20 @@ def analyze(self, line):
                 self._saved.add(pin)
             return self.Continue()
 
+        match = re.search(
```

Comment:
This is the new PEP-691 interaction needed for Pip 22.2.2 since it no longer logs hashes along with URLs. The vendored Pip never triggers this code path because it does not log this "Fetched page ..." line.